smart-contracts/contracts/PublicLock.sol

```diff
@@ -52,6 +52,8 @@ contract PublicLock is
     public
     MixinLockCore(_owner, _expirationDuration, _keyPrice, _maxNumberOfKeys, _version)
   {
+    // registering the interface for erc721 with ERC165.sol
```

Comment:
Another magic number.  Not a fan, but that's how it goes =p

Could you maybe change to something like
`registering the interface for erc721 with ERC165.sol using the ID specified in the standard: https://eips.ethereum.org/EIPS/eip-721`

Reply:
Sure, will do.